develop/docker-compose/docker-compose.yml

```diff
@@ -76,7 +76,7 @@ services:
     volumes:
       - ./grafana/provisioning/:/etc/grafana/provisioning/
   temporal-ui:
-    image: temporalio/ui:2.26.2
+    image: temporalio/ui:latest
```

Comment:
`latest` is usually not the best idea; but I think here it works great because we won't have to remember updating this - and the risk for breaking changes seems low to me.